tests/components/rfxtrx/test_init.py

```diff
@@ -55,21 +58,19 @@ async def test_invalid_config(hass):
 
 async def test_fire_event(hass, rfxtrx):
     """Test fire event."""
-    assert await async_setup_component(
-        hass,
-        "rfxtrx",
-        {
-            "rfxtrx": {
-                "device": "/dev/serial/by-id/usb"
-                + "-RFXCOM_RFXtrx433_A1Y0NJGR-if00-port0",
-                "automatic_add": True,
-                "devices": {
-                    "0b1100cd0213c7f210010f51": {"fire_event": True},
-                    "0716000100900970": {"fire_event": True},
-                },
-            }
+    entry_data = create_rfx_test_cfg(
+        device=("/dev/serial/by-id/usb" + "-RFXCOM_RFXtrx433_A1Y0NJGR-if00-port0"),
```

Comment:
Please use f-strings instead of string concatenation.

Reply:
I just blindly copied this, we don't even need f-strings. It's just two fixed strings with no variables added so I reduces it two one string.